portfolio.css

```diff
@@ -20,765 +90,601 @@
+}
 
-  .social-icons-2 {
-    width: 6vw;
-    height: 6vw;
-    margin-right: 3vw;
-    margin-left: 3vw;
-  }
+#form-grid {
+  display: grid;
+  grid-template-columns: 100%;
+  align-items: center;
+  justify-content: center;
+  text-align: center;
+  font-weight: bold;
+  font-size: 40px;
+  line-height: 52px;
+  margin-left: 3vh;
+  margin-right: 3vh;
+  margin-top: 10vh;
+}
 
-  .email {
-    font-family: 'Inter', sans-serif;
-    font-weight: normal;
-    font-size: 4vw;
-    line-height: 7vw;
-    color: #42526e;
-    height: 30vh;
-    text-align: center;
-    margin-top: 1vh;
-  }
+#form-grid input {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 8vh;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #d0d9d4;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-top: 4vh;
+  padding: 3vw;
+  font-family: 'Inter', sans-serif;
+  color: #6b778c;
+  font-weight: normal;
+  font-size: 3.3vw;
+  line-height: 24px;
+}
 
-  .light {
-    display: flex;
-    align-items: flex-end;
-    margin-top: 20vh;
-    width: 100%;
-  }
+#form-grid textarea {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  height: 19vh;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #60c095;
+  box-sizing: border-box;
+  border-radius: 4px;
+  margin-top: 4vh;
+  font-family: 'Inter', sans-serif;
+  font-weight: normal;
+  font-size: 3.5vw;
+  line-height: 24px;
+  color: #091e42;
+  padding: 3vw;
+  resize: initial;
+  background-image: url(images/comment-icon.svg);
+  background-repeat: no-repeat;
+  background-position-x: right;
+  background-position-y: bottom;
 }
 
-@media (max-width: 375px) {
-  * {
-    box-sizing: border-box;
-    margin: 0;
-    padding: 0;
-  }
+#contact-grid {
+  display: grid;
+  grid-template-columns: 100%;
+  align-items: center;
+  justify-content: center;
+}
 
-  header {
-    width: 100%;
-  }
+.B-2 {
+  display: flex;
+  justify-content: center;
+  align-items: flex-end;
+  width: 100%;
+  height: 10vh;
+  margin-top: 5%;
+}
 
-  li {
-    list-style: none;
-  }
+.social-media-2 {
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  margin-top: 6vw;
+}
 
-  a {
-    text-decoration: none;
-  }
+.social-icons-2 {
+  width: 6vw;
+  height: 6vw;
+  margin-right: 3vw;
+  margin-left: 3vw;
+}
 
-  .page-container {
-    display: flex;
-    flex-direction: column;
-  }
+.email {
+  font-family: 'Inter', sans-serif;
+  font-weight: normal;
+  font-size: 4vw;
+  line-height: 7vw;
+  color: #42526e;
+  height: 30vh;
+  text-align: center;
+  margin-top: 1vh;
+}
 
-  .header-container {
-    display: flex;
-    justify-content: space-between;
-    align-items: center;
-    width: 100%;
-    height: 50px;
-  }
+.light {
+  display: flex;
+  align-items: flex-end;
+  margin-top: 20vh;
+  width: 100%;
+}
 
-  .Nav-1 a {
-    color: #28352f;
-    font-size: 5.5vw;
-    padding-left: 5vw;
+@media (min-width: 991.99px) {
```

Comment:
- [ ] The value for min-width property should be in range of 992px - 1100px. These are the accepted desktop screen breakpoints. Change your value. What you are using lies under the tablet screen sizes.